Last-Update: 2018-06-09
Forwarded: not-needed
Author: Dmitry Smirnov <onlyjob@debian.org>
Description: fix import path [BROKEN]
 Docker bundles modified version of "github.com/Nvveen/Gotty".
 This patch causes "panic: runtime error: index out of range"
 on `docker (pull|build)`.

--- a/engine/pkg/jsonmessage/jsonmessage.go
+++ b/engine/pkg/jsonmessage/jsonmessage.go
@@ -7,9 +7,9 @@
 	"os"
 	"strings"
 	"time"
 
-	gotty "github.com/Nvveen/Gotty"
+	gotty "github.com/nvveen/gotty"
 	"github.com/docker/docker/pkg/term"
 	units "github.com/docker/go-units"
 )
 
